easybuild/easyconfigs/x/XML-LibXML/XML-LibXML-2.0207-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'Bundle'
+
+name = 'XML-LibXML'
+version = '2.0207'
+
+homepage = 'https://metacpan.org/pod/distribution/XML-LibXML/LibXML.pod'
+description = "Perl binding for libxml2"
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+builddependencies = [
+    ('pkgconf', '1.8.0'),
+    ('binutils', '2.37'),
+]
+
+dependencies = [
+    ('Perl', '5.34.0'),
+    ('libxml2', '2.9.10'),
+]
+
+exts_defaultclass = 'PerlModule'
+exts_filter = ("perldoc -lm %(ext_name)s ", "")
+
+exts_list = [
+    ('File::chdir', '0.1010', {
+        'source_tmpl': 'File-chdir-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN'],
+        'checksums': ['efc121f40bd7a0f62f8ec9b8bc70f7f5409d81cd705e37008596c8efc4452b01'],
+    }),
+    ('Alien::Base', '2.46', {
+        'source_tmpl': 'Alien-Build-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/P/PL/PLICEASE/'],
+        'checksums': ['4bb301fa2b6296a9d8e285000932ea3340832a3a70bdb9c5e1a63c6fd1cf4a67'],
+    }),
+    ('Alien::Libxml2', '0.17', {
+        'source_tmpl': 'Alien-Libxml2-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/P/PL/PLICEASE'],
+        'checksums': ['73b45244f0b5c36e5332c33569b82a1ab2c33e263f1d00785d2003bcaec68db3'],
+    }),
+    ('XML::LibXML', version, {
+        'source_tmpl': 'XML-LibXML-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/'],
+        'checksums': ['903436c9859875bef5593243aae85ced329ad0fb4b57bbf45975e32547c50c15'],
+    }),
+]
+
+modextrapaths = {
+    'PERL5LIB': 'lib/perl5/site_perl/%(perlver)s/',
+}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/perl5/site_perl/%(perlver)s/x86_64-linux-thread-multi/XML/LibXML'],
```

Comment:
I'll update this to avoid the hardcoded `x86_64`...

On POWER, this should be: `ppc64le-linux-thread-multi`
On ARM64, this should be: `aarch64-linux-thread-multi`

I guess it's worth doing the same in the existing (sufficiently recent) easyconfigs for `XML-LibXML` too...

Reply:
Using `%(arch)s` works on the x86_64 and POWER I have available.